CONTRIBUTING.md

```diff
@@ -186,34 +186,78 @@ This often provides additional considerations and avoids unnecessary work.
    the PR as a draft first and mark it as ready for review after the feedbacks
    from the continuous integration (CI) system or any required fixes.
 
-### Special Considerations for Contributing External Tests
+### Contributing Internal Tests
+
+In `py-dcqc`, any test where the primary business logic is executed within the package itself is considered internal. One example is the `Md5ChecksumTest`.
+
+When contributing an internal test be sure to do the following:
+
+1. Follow the steps above to set up `py-dcqc` and create your contribution.
+
+1. Include a class docstring that describes the purpose of the test.
+
+1. Include the following class attributes:
+
+   - `tier`: An `int` value describing the complexity of the test contributed. `tier` values include:
+     - `1`: File Integrity
```

Comment:
nit: It might be clearer within the test if these are an enum (That could map to an integer). That way instead of just seeing:
```
    tier = 1
```

You will see:
```
    tier = SUBJECTIVE_CONFORMANCE
```

Then when evaluating it in the base class check the enum or int value of the attribute. The tier type would be:
```
tier: Union[int, MyTierEnum]
or enforce tier to be your enum:
tier: MyTierEnum
```